.github/workflows/unit-integration-tests.yml

```diff
@@ -109,6 +109,18 @@ jobs:
         ports:
           - 8125:8125/udp
           - 8126:8126
+      testagent:
+        image: ghcr.io/datadog/dd-apm-test-agent/ddapm-test-agent:latest
```

Comment:
Using the `latest` version can be useful to avoid manual updates but also exposes us to breaking changes and bugs, i.e a small bug the test agent can make the CI fail and it impacts 4+ teams (the teams that actively contributes to dd-trace-go).

So my suggestion here is either we use a pinned version of the test agent, or use latest but mark the jobs as allowed to fail.

Alternatively, it should be fine to keep `latest` if you guarantee that a new test agent version can only be released if these tests run and pass in the test agent repo as well.

Reply:
good idea, i pinned the version